requirements_dev.txt

```diff
@@ -6,4 +6,4 @@ pytest-mock==3.10.0
 pytest-raises==0.11
 pytest-socket==0.5.1
 pytest-sugar==0.9.5
-pytest-xdist==2.5.0
+pytest-xdist==3.0.2
```

Comment:
Apparently pytest-xdist is now in the constraints file: https://raw.githubusercontent.com/apache/airflow/constraints-2.4.1/constraints-3.10.txt

Looks like we'll just need to unpin this dependency entirely so it can be managed by the constraints file.